config/v1/types_infrastructure.go

```diff
@@ -536,6 +536,14 @@ type AWSPlatformStatus struct {
 	// +optional
 	// +nullable
 	CloudLoadBalancerConfig *CloudLoadBalancerConfig `json:"cloudLoadBalancerConfig,omitempty"`
+
+	// ipFamily indicates the IP Address family supported by cluster nodes.
```

Comment:
Explain the valid values and what they mean as part of the godoc

Reply:
Done.